pc-bufsw.el

```diff
@@ -210,6 +210,13 @@ Formatting can be added using text properties, e.g.:
     :type '(choice (const nil)
                    face))
 
+  (defcustom pc-bufsw-prefer-current-window nil
```

Comment:
This should not be an individual option but rather another case in pc-bufsw-other-windows.

Reply:
OK, so, `nil`, `:skip`, and `:sort`? And, convert `pc-bufsw-prefer-current-frame` to `pc-bufsw-other-windows` with the same options?

Reply:
Yes. :sort is nice - brief and precise.

Reply:
Sorry, I misremembered what `pc-bufsw-other-windows` did.

`pc-bufsw-other-windows` filters buffers *currently* being displayed in *other* windows. `pc-bufsw-prefer-current-window` sorts buffers that have been *previously* displayed in the *current* window.

These two seem orthogonal, and I don't see an obvious way to merge them.

What we could do is, for consistency, have `pc-bufsw-other-windows`, `pc-bufsw-current-window`, and `pc-bufsw-current-frame`, give all three the same options (`nil` / `:sort` / `:skip`), and introduce `pc-bufsw-other-window-face` OSLT.

Reply:
I suggest just have pc-bufsw-current-window with nil :sort and :skip and remove pc-bufsw-other-windows. I pushed a version with the latter in December, so I doubt anybody is using it. I think if i was aware about history lists for buffers I would most likely implemented it with the same semantic as pc-bufsw-current-window  :skip. And with all your nice changes it make sense to bump the version to 4.0.

Reply:
Also, pc-bufsw-current-window should have nil, :sort and :only as values, not :skip, to reflect semantics.